booster/library/Booster/SMT/Interface.hs

```diff
@@ -168,13 +168,13 @@ getModelFor ::
     Map Variable Term -> -- supplied substitution
     io (Either SMTError (Either SMT.Response (Map Variable Term)))
 getModelFor ctxt ps subst
-    | null ps && Map.null subst = Log.withContext "smt" $ do
-        Log.logMessage ("No constraints or substitutions to check, returning Sat" :: Text)
+    | null ps && Map.null subst = Log.withContext Log.CtxSMT $ do
+        Log.logMessage ("No Constraints Or Substitutions To Check, Returning Sat" :: Text)
```

Comment:
Such Capitalism!

Reply:
:rofl: Emacs shortcuts going rogue.... I remember pressing the wrong keys (`M^c`) here... corrected everything that the compiler cares about but not the string contents :smile:

Reply:
```suggestion
        Log.logMessage ("No constraints or substitutions to check, returning Sat" :: Text)
```